content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -228,7 +228,7 @@ kubectl rollout restart deployment/frontend                      # Rolling resta
 
 cat pod.json | kubectl replace -f -                              # Replace a pod based on the JSON passed into std
 
-# Force replace, delete and then re-create the resource. Will cause a service outage.
+# Force replaces, delete and then re-create the resource. Will cause a service outage.
```

Comment:
This sentence is not grammatically correct. I'm wondering what's the purpose of the changes though.

Reply:
Hi @andrzejsydor .
This original comment is confusing.
How about updating the comment to:
```
# `kubectl replace --force` deletes then recreates the resource. You can expect a service outage.
```